modules/core/arrow-core-data/src/test/kotlin/arrow/core/SortedMapKTest.kt

```diff
@@ -23,6 +31,11 @@ class SortedMapKTest : UnitSpec() {
       fix()["key"] == b.fix()["key"]
   }
 
+  val EQK = object : EqK<SortedMapKPartialOf<String>> {
+    override fun <A> Kind<SortedMapKPartialOf<String>, A>.eqK(other: Kind<SortedMapKPartialOf<String>, A>, EQ: Eq<A>): Boolean =
+      SortedMapK.eq(String.eq(), EQ).run { this@eqK.fix().eqv(other.fix()) }
+  }
+
```

Comment:
Can this be added as an instance for SortedMapK directly?

Reply:
It can and we should :)

Reply:
Although the `Eq` instance for `SortedMapK` should be parameterized for both the keys and values.

Reply:
i have added an Align instance

Reply:
> Although the Eq instance for SortedMapK should be parameterized for both the keys and values.

@nomisRev i do not understand what you exactly mean by this? could you explain a bit more please?

Reply:
We can add an instance where `K` remains generic.

```
interface SortedMapK<K> : EqK<SortedMapKPartialOf<K>> {
   fun EQK(): Eq<K>
   
    override fun <A> Kind<SortedMapKPartialOf<String>, A>.eqK(other: Kind<SortedMapKPartialOf<String>, A>, EQ: Eq<A>): Boolean =
      SortedMapK.eq(EQK(), EQ).run { this@eqK.fix().eqv(other.fix()) }
}
```

Reply:
ok now i've got it. didn't get your comment was related to

> Can this be added as an instance for SortedMapK directly?

thanks!

Reply:
done